transport/server.cc

```diff
@@ -1218,6 +1218,10 @@ std::unique_ptr<cql_server::response>
 cql_server::connection::make_result(int16_t stream, shared_ptr<messages::result_message> msg, const tracing::trace_state_ptr& tr_state, bool skip_metadata)
 {
     auto response = std::make_unique<cql_server::response>(stream, cql_binary_opcode::RESULT, tr_state);
+    if (!msg->warnings().empty()) {
```

Comment:
I'm not sure if it really matters here, but it looks like a nice candidate for using `__builtin_expect(..., false)` here, since we expect most responses to not produce warnings and this code is in the hot path.

Reply:
Done